src/substituteteacher.js

```diff
@@ -1,4 +1,6 @@
-(function(window) {
+/* */
+"format cjs";
```

Comment:
Technically you don't have to specify it directly, most tools will auto-detect or default to it, but, it's better to [be explicit](https://github.com/jspm/registry/wiki/Configuring-Packages-for-jspm#package-module-format), also with this pull you're supporting CJS, but also allowing AMD and global export so this avoids any conflicts I guess.